### DISCUSSION TITLE
Supporting Next.js 14 ?

### DISCUSSION THREAD

**Discussion by sadeghnakhjavani** [chapter-three/next-drupal#734] [Q&A]
Hello and thank you for your great solution to mix a powerful CMS like Drupal with a new UI technology like Next.js and other new and widely used tools like Tailwind.

1. My first question is about supporting Next.js 14. Does your solution support Nex.js 14 and it's new features?
2. The second question is about Request Memoization feature. This feature only applies to the GET method in fetch requests.
Does the DrupalClient class use this method to get data underthehood?

**Comment by zma-athena** [comment]
I think the support Next.js 14 feature was completed in beta version, see https://github.com/chapter-three/next-drupal/issues/578
